tests/languages/purebasic/comment_feature.test

```diff
@@ -0,0 +1,13 @@
+;
+;gugug du Affe

```

Comment:
Maybe a slightly less aggressive example?

Reply:
Well... yes, this seems to be a forgotten corpse from all my tests, although I wouldn't name this aggressive?!
We can change it to something like
;This is a comment.

btw.:
The test procedure for comments drove me crazy!
I wanted to add a test for comments BEHIND a legal code line.
This seems to be impossible, and the error messages are also quite useless (at least for me)

Reply:
> The test procedure for comments drove me crazy!

What happened here? I don't see how this can cause an error? What code did you try to test?
(Just a guess: Did you get a JSON parse/syntax error?)

Reply:
I'm quite sure I misunderstood something, but from my understanding, this here should be fine:
```
a$ = 2 ;Test

----------------------------------------------------

[
	["comment", ";Test"]
]

----------------------------------------------------

Checks for comments.
```
The comment part (to test) is ;Test, so what is wrong here?

Here the (cryptic) error message, which drove me crazy:

```
: expected [ Array(4) ] to deeply equal [ [ 'comment', ';Test' ] ]
      + expected - actual

       [
      -  "a$ "
         [
      -    "operator"
      -    "="
      -  ]
      -  [
      -    "number"
      -    "2"
      -  ]
      -  [
           "comment"
           ";Test"
         ]
       ]
```

Reply:
That's the diff of the expected token stream (your JSON) and the actual token stream Prism output when highlighting `a$ = 2 ;Test`. (But yes, the diff isn't helpful at all here. It can be useful when there are only minor differences but I usually ignore it.)

Your JSON has to be the token stream of the entire code snippet. That includes whatever tokens `a$ = 2` will be.

I get that this can be confusing. I'll add it to the [doc](https://prismjs.com/test-suite.html).